app/helpers/custom_form_error_helper.rb

```diff
@@ -0,0 +1,19 @@
+module CustomFormErrorHelper
+  def errors(object, attribute)
+    error_messages = object.try(:errors).try(:full_messages_for, attribute)
+
+    if error_messages.is_a?(Array) && error_messages.count.positive?
```

Comment:
__[Reek]__ [FeatureEnvy] CustomFormErrorHelper#errors refers to 'error_messages' more than self (maybe move it to another class?) ([link](https://sider.review/gh/repos/47168038/pulls/585#issue-4232502))

<sub>posted by Sider</sub>